admin/JDG_HotRod_Data_Sources.adoc

```diff
@@ -101,20 +162,79 @@ To enable the Hotrod client in the resource-adapter to communicate using SSL, co
 
-The following are the additional properties that need to be configured if using the Remote Cache for external materialization:
+==== *Enable using Hot Rod Authentication*
+
+This section describes how to configure the JDG resource-adapter to utilize the Hot Rod Client authentication.
+
+The JDG server will need to have its _hotrod-connector_ configured to enable accessing using authentication. 
+
+[source,xml]
+.*Example configured hotrod-connector*
+----
+<hotrod-connector socket-binding="hotrod" cache-container="default">
+   <authentication security-realm="ApplicationRealm">
+      <sasl server-name="myhotrodserver" mechanisms="DIGEST-MD5" qop="auth" />
+   </authentication>
+</hotrod-connector>
+----
+
+Note the following
+
+* server-name attribute - it is the name that the server declares to incoming clients and therefore the client configuration must match the <<AuthServerName>> property.
+* security-realm - it should be the same one defined for <<AuthApplicationRealm>>
+* mechanisms - it should be the same one defined for <<AuthSASLMechanism>>
+
+
+The following properties define how the JDG resource-adapter can be configured to talk to the JDG server based on the _hotrod_connector_.
 
 |===
-|Property Name |Req. |Description
+|Property Name |Description
+
+| AuthServerName | The name declared as the _server_name_ in the JDG Server
+| AuthApplicationRealm | The security realm defined in the JDG server.
+| AuthSASLMechanism   | The SASL _mechanisms_ (e.g., PLAN, DIGEST-MD5, etc.)
+|===
+
+The Hot Rod protocol supports authentication leveraging different SASL mechanisms. The supported SASL mechanisms (usually shortened as mechs) are.
+
+* PLAIN - This is the most insecure mech, since credentials are sent over the wire in plain-text format, however it is the simplest to get to work. In combination with encryption (i.e. SSL) it can be used safely
+* DIGEST-MD5 - This mech hashes the credentials before sending them over the wire, so it is more secure than PLAIN
+* GSSAPI - This mech uses Kerberos tickets, and therefore requires the presence of a properly configured Kerberos Domain Controller (such as Microsoft Active Directory)
+* EXTERNAL - This mech obtains credentials from the underlying transport (i.e. from a X.509 client certificate) and therefore requires encryption using client-certificates to be enabled.
+
+
+If using Hot Rod Authentication, then the following _admin_ properities will need to be defined so that metadata can be obtained without having to have a user execute a query.
 
-| StagingCacheName | Y | Cache name for the staging cache used in materialization 
-| AliasCacheName   | Y | Cache name for the alias cache used in tracking aliasing of the caches used in materialization.  This cache can be shared with other configured materializations.
 |===
+|Property Name |Description
+
+| AdminUserName | Username for administration operation
```

Comment:
Is it needed?  yes, as indicated - "If using Hot Rod Authentication, then the following _admin_ properities will need to be defined so that metadata can be obtained without having to have a user execute a query."

Which means using any of the JDG authentication options.  Which at this point, there are only 2:  username/password or subject.

This is needed because the metadata can be triggered (retrieved) at a different time when no actual user has submitted a query.